src/Microsoft.AspNet.Mvc.Core/CacheProfile.cs

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.Mvc
+{
+	public class CacheProfile
+	{
+        public CacheProfile(string name)
+        {
+            Name = name;
+        }
+
+        /// <summary>
+        /// Gets or sets the name of the CacheProfile.
+        /// The CacheProfile is referred in the ResponseCache attribute using this name.
```

Comment:
<see cref="ResponseCacheAttribute" />, in fact I think it should be an interface rather than a concrete type.